src/plugins/trace/exe/Program.cs

```diff
@@ -234,7 +234,7 @@ static void Main(string[] args)
                     Console.Write("quictrace> ");
 
                     var input = Console.ReadLine();
-                    if (input == "--exit" || input == "exit" || input == "-e")
+                    if (input == null || input == "--exit" || input == "exit" || input == "-e")
```

Comment:
Does `null` mean "User hit enter" or "User hit ctrl+c"?

Reply:
Null means end of file. So if user passes in a file through stdin null will be returned after the last line. Always been able to be return, but 5.0 properly annotated Console.ReadLine.

Reply:
Ok. I just want to make sure "User hit enter" doesn't exit.

Reply:
Nope. That will return an empty string.